### ISSUE TITLE
Add maskrcnn_resnet50_fpn_v2 from TorchVision models

### ISSUE THREAD

**Issue by medphisiker** [airctic/icevision#1167]
## 🚀 Feature

Hello.

there is new intresting version of **Masked RCNN** model in TorchVision ([link](https://pytorch.org/vision/stable/models.html#instance-segmentation)).
**maskrcnn_resnet50_fpn_v2** - Improved Mask R-CNN model with a ResNet-50-FPN backbone from the [Benchmarking Detection Transfer Learning with Vision Transformers](https://arxiv.org/abs/2111.11429) paper.

maskrcnn_resnet50_fpn_v2 model gives effective increase([link](https://pytorch.org/vision/stable/models.html#instance-segmentation)) for MS COCO metric in comparision with classic **maskrcnn_resnet50_fpn**.

![image](https://user-images.githubusercontent.com/4876436/213477454-f7433bcd-a517-436b-be6e-3ee2ac7404a0.png)

I see some examples of fine tuning. The code for fine tuning **maskrcnn_resnet50_fpn_v2** and **maskrcnn_resnet50_fpn** are identical.
Ice Vision framework has support for classic TorchVision's **maskrcnn_resnet50_fpn** fine tuning. It will be great if Ice Vision framework also has support for new TorchVision's **maskrcnn_resnet50_fpn_v2**.

**Describe the solution you'd like**
It will be great if Ice Vision framework also has support for new TorchVision's **maskrcnn_resnet50_fpn_v2**. Also there is an updated version of the these detectors, - **FasterRCNN_ResNet50_FPN_V2** and **RetinaNet_ResNet50_FPN_V2**.

![image](https://user-images.githubusercontent.com/4876436/213479643-5adbf44d-3855-45f5-b948-30ab884341d8.png)

**Describe alternatives you've considered**
Currently, we already have many excellent neural networks for detection in the Ice Vision framework. But it is worth noting that **Faster and Masked RCN** are multi-stage detectors. Most of the more accurate detectors presented in the framework are single-stage. 
In one competition, I used **YOLOv7**, which had a higher metric on MS COCO for detection. But the competitors that used the classic multistage **Faster R-CNN** won. It turned out that on a dataset with crowded objects, **Faster RCNN** works better than a single-stage **YOLOv7**, even though there is a big difference in metrics on MS COCO in the YOLOv7 slider.
